docs/generated/packages/cypress/generators/cypress-component-project.json

```diff
@@ -19,7 +19,8 @@
         "type": "string",
         "description": "The name of the project to add cypress component testing to",
         "$default": { "$source": "projectName" },
-        "x-prompt": "What project should we add Cypress component testing to?"
+        "x-prompt": "What project should we add Cypress component testing to?",
+        "x-priority": "important"
```

Comment:
same here, `project` is already required. 
I'm going to not post the same comment for other `required` fields